### ISSUE TITLE
Auto-generate pretty cover

### ISSUE THREAD

**Issue by falcaopetri** [CorrectRoadH/logseq-gallery#3]
Hi. First of all, thanks for the amazing plugin!

I was wondering about the cover shown in the cards like "Marketing" and "Indie Hacker", which are composed by the page's title + solid color.

<img src="https://github.com/CorrectRoadH/logseq-gallery/blob/master/imgs/screenshot-1.png?raw=true" height="200">

I guess (mainly based on [note's render logic](https://github.com/CorrectRoadH/logseq-gallery/blob/3c5a90a836e500d3c4c2d1c687529093a0fc4e63/src/Gallery.tsx#L84-L161)) that these covers were somewhat (manually?) generated beforehand. 

Do you have an easy way to generate these "title+background color" covers?

Would it be possible to have this automatically generated by this plugin?

And then have it offered as an option to replace "No cover" or even the default markdown preview? (a little bit like Card preview option in Notion)

**Comment by CorrectRoadH** [comment]
Yes. These covers are generated manually. It would be a good idea to provide a way to automate them.🤔But I'm not sure when I'll be free to work on it. If anyone interested, feel free to submit a PR.
